includes/Experiments.php

```diff
@@ -387,17 +387,6 @@ public function get_experiments() {
 				'description' => __( 'Enable post locking', 'web-stories' ),
 				'group'       => 'general',
 			],
-			/**
-			 * Author: @miina
-			 * Issue: #7055
-			 * Creation date: 2021-04-09
-			 */
-			[
-				'name'        => 'customPageTemplates',
-				'label'       => __( 'Custom Page Templates', 'web-stories' ),
-				'description' => __( 'Enable adding custom page templates', 'web-stories' ),
-				'group'       => 'editor',
-			],
```

Comment:
Nit: I usually advocate for first setting `default => true` in a PR like this, and then do the complete removal of `useFeatures` etc. in a separate PR.

This makes reverting easier should there be any significant errors.

Reply:
Hm, alright, will add it back just in case.